queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Kindly replace the 2nd query with this as it should be written in this way. 
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`